resources/scripts/setupDB.js

```diff
@@ -6,6 +6,16 @@
             place.alternatenames = '';
         }
 
+        //fuck string with "Nan" code
+        Object.keys(place).forEach(function (key) {
+            if (typeof place[key] != "string" &&
+                typeof place[key] != "array" &&
+                typeof place[key] != "object" &&
+                isNaN(place[key])) {
```

Comment:
http://adripofjavascript.com/blog/drips/the-problem-with-testing-for-nan-in-javascript.html

>Currently the best way to check whether a value is NaN is to check whether it is not equal to itself. If so, then you know that it is NaN.